src/include/sof/audio/component.h

```diff
@@ -282,7 +282,8 @@ struct comp_ops {
 	/**
 	 * Creates a new component device.
 	 * @param drv Parent component driver.
-	 * @param comp Component parameters.
+	 * @param ipc_config Component parameters.
+	 * @param spec Pointer to initialization data
```

Comment:
I was expecting parameter-name to match the C signature (i.e. "ipc_specific_config"), but I guess it doesn't matter (either to doxygen or the person consuming the docs).

Reply:
force of habit, every single module calls this arg spec in their implementation. I am going to update the signature to match that.